### ISSUE TITLE
Account creation via API

### ISSUE THREAD

**Issue by simison** [vedmaka/Mediawiki-reCaptcha#4]
Could be worth mentioning at the readme that this extension doesn't support account creation via API and will only respond:
```xml
<?xml version="1.0"?>
<api>
  <error code="createaccount-hook-aborted" info="Failed to bypass robots check" xml:space="preserve"></error>
</api>
```

FYI, here's how to disable account creation via API, put this to your `LocalSettings.php`:
```php
$wgAPIModules['createaccount'] = 'ApiDisabled';
```

I'm not sure if new reCaptcha even would support building something like that, but anyway here's how the response from ConfirmEdit looks like: https://www.mediawiki.org/wiki/API:Account_creation#CAPTCHA

**Comment by vedmaka** [comment]
Yep, i thought about it. I am not sure, but looks like new reCaptcha not supporting workflow with raw image/code. I tested a bit "nojs" way (https://developers.google.com/recaptcha/docs/faq#no_js) but with no success.

**Comment by simison** [comment]
Aight!

I noticed ConfirmEdit dudes are in the progress of testing new reCaptcha for their plugin, so once that gets done I'm sure majority will be moving to use that instead of this one.
